Makefile

```diff
@@ -378,8 +378,8 @@ package-capi:
 	cp lib/c-api/wasm.h* package/include
 	cp lib/c-api/README.md package/include/README.md
 ifeq ($(OS), Windows_NT)
-	cp target/release/wasmer_c_api.dll package/lib
-	cp target/release/wasmer_c_api.lib package/lib
+	cp target/release/wasmer_c_api.dll package/lib/libwasmer.dll
```

Comment:
AFAIR on Windows we should not add the `lib` prefix in file names.

Reply:
Correct! Fixed.